docs/docs_skeleton/vercel.json

```diff
@@ -2509,9 +2509,21 @@
       "destination": "/docs/integrations/memory/cassandra_chat_message_history"
     },
     {
-      "source": "/docs/modules/memory/integrations/cassandra_chat_message_history",
+      "source": "/en/latest/modules/memory/examples/cassandra_chat_message_history.html",
```

Comment:
why this change? isn't this duplicate of above

Reply:
Oops. PyCharm automatically refactored it. My bad. I had to verify it anyway.